tests/check/check.go

```diff
@@ -520,9 +529,10 @@ var matrix = individualTest{
 			},
 			CheckAssertions: []*checktest.Assertion{
```

Comment:
**Question:** `CheckAssertions` doesn't really make sense anymore, right? It may trigger a bigger change but could be appropriate.

Reply:
True. It's a big change though, i'd prefer to do it in a different PR